README.md

```diff
@@ -19,3 +19,32 @@ This repository contains machine-readable specifications for the auxillilary ser
 - [ ] Exhaustive RPC testing for Forest docker image.
 - [ ] Sync testing for PRs.
 - [ ] Export testing for PRs.
+
+## Running Forest In DigitalOcean
+
+## Architectural Implementation
+
+![Untitled Diagram drawio (8)](https://user-images.githubusercontent.com/47984109/215227510-dac5b8fb-8019-4388-a0e7-d5c432b95d70.png)
+
+The flow goes from Terraform for provisioning of the servers and Anisble to run all neccessary installations including forest.
+
+## Requirements 
+- RAM: 32GB
+- VCPU: 8
+- Startup Disk Size: 100 GB
```

Comment:
For startup, we need at least 200G, +100G to download the snapshot and another +100G to fill the DB. After this process, we can delete the snapshot.